debian/source/format

```diff
@@ -1 +1 @@
-3.0 (native)
+3.0 (quilt)
```

Comment:
What speaks against native? I felt like an apt transport was a good use case for native because it's unlikely to be used outside of Debian (or a Debian derivative), so there is no need to distinguish between upstream and downstream releases. @h01ger can you shine a light here?

Reply:
If native, no need finally to make all I've made for branches. I would just need to revert back `quilt` to `native`.